x-pack/plugins/observability_solution/slo/public/pages/slo_edit/components/common/data_preview_chart.tsx

```diff
@@ -339,7 +341,7 @@ export function DataPreviewChart({
                 yAccessors={['value']}
                 data={(previewData ?? []).map((datum) => ({
                   date: new Date(datum.date).getTime(),
-                  value: datum.sliValue >= 0 ? datum.sliValue : null,
+                  value: datum.sliValue && datum.sliValue >= 0 ? datum.sliValue : null,
```

Comment:
Do you think we can have negative value or the check was just in place for the `-1` (no data) special case?